src/editor/api/scene.js

```diff
@@ -189,6 +191,101 @@ const checkIfImagePathIsEmpty = async (sceneId) => {
   }
 };
 
+const saveScreenshot = async (value) => {
+  const screenshotEl = document.getElementById('screenshot');
+  screenshotEl.play();
+
+  if (value === 'img') {
+    screenshotEl.setAttribute(
+      'screentock',
+      'imgElementSelector',
+      '#screentock-destination'
+    );
+  }
+
+  posthog.capture('screenshot_taken', {
+    type: value,
+    scene_id: STREET.utils.getCurrentSceneId()
+  });
+
+  screenshotEl.setAttribute('screentock', 'type', value);
+  screenshotEl.setAttribute('screentock', 'takeScreenshot', true);
+};
+
+const uploadThumbnailImage = async () => {
+  try {
+    // saveScreenshot('img');
```

Comment:
note, i commented this out, since in every call `makeScreenshot` is called prior